src/App.js

```diff
@@ -41,6 +40,19 @@ class App extends Component {
     this.setState({items})
   }
 
+  handleSubmit(task) {
+    console.log("what is task", task);
```

Comment:
Don't forget to remove console.logs when submitting a PR for the sake of good practice. 👍